frontend/src/components/ShootHibernation/GShootActionChangeHibernation.vue

```diff
@@ -5,49 +5,44 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <template>
-  <g-shoot-action-dialog
-    v-if="dialog"
+  <g-action-button-dialog
     ref="actionDialog"
     :shoot-item="shootItem"
+    width="600"
     :caption="caption"
+    :text="buttonText"
     :confirm-button-text="confirmText"
-    :confirm-required="confirmRequired"
-    width="600"
-  >
-    <template v-if="!isShootSettingHibernated">
-      This will scale the worker nodes of your cluster down to zero.<br><br>
-      Type <strong>{{ shootName }}</strong> below and confirm to hibernate your cluster.<br><br>
-    </template>
-    <template v-else-if="hasShootWorkerGroups">
-      This will wake up your cluster and scale the worker nodes up to their previous count.<br><br>
-    </template>
-    <template v-else>
-      This will wake up your cluster.<br><br>
-    </template>
-  </g-shoot-action-dialog>
-  <g-shoot-action-button
-    v-if="button"
-    ref="actionButton"
-    :shoot-item="shootItem"
+    confirm-required
     :disabled="!isHibernationPossible && !isShootSettingHibernated"
     :icon="icon"
-    :text="buttonText"
-    :caption="caption"
-    @click="internalValue = true"
-  />
+    @dialog-opened="onConfigurationDialogOpened"
+  >
+    <template #scrollable-content>
+      <v-card-text>
+        <template v-if="!isShootSettingHibernated">
+          This will scale the worker nodes of your cluster down to zero.<br><br>
+          Type <strong>{{ shootName }}</strong> below and confirm to hibernate your cluster.<br><br>
+        </template>
+        <template v-else-if="hasShootWorkerGroups">
+          This will wake up your cluster and scale the worker nodes up to their previous count.<br><br>
+        </template>
+        <template v-else>
+          This will wake up your cluster.<br><br>
+        </template>
+      </v-card-text>
+    </template>
+  </g-action-button-dialog>
 </template>
 
 <script>
-import GShootActionButton from '@/components/GShootActionButton.vue'
-import GShootActionDialog from '@/components/GShootActionDialog.vue'
+import GActionButtonDialog from '@/components/dialogs/GActionButtonDialog.vue'
 
 import { shootItem } from '@/mixins/shootItem'
 import { errorDetailsFromError } from '@/utils/error'
 
 export default {
   components: {
-    GShootActionButton,
-    GShootActionDialog,
+    GActionButtonDialog,
```

Comment:
Here we still use `notify`. In an other component we have replaced it with `setSuccess`. What is the way to go?

Reply:
we use setSuccess / setAlert everywhere so I changed the remaining component to also use setSuccess